role-name/molecule/roles/molecule.yml

```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: centos8
+    image: centos:8
+    pre_build_image: true
+lint:
+  'set -e ansible-lint'
```

Comment:
The `set -e` command isn't strictly required here as the exit code of the last command will be used and given that we're only executing the single command `ansible-lint`. Having said that, if roles created from this template need to extend the linting stage by introducing additional commands we would want the first command that produces a non-zero exit code in the context of a list of commands to be used. I'd suggest we keep `set -e`, but use the pipe character to indicate a block-style literal:

```yaml
lint: |
  set -e
  ansible-lint
```

This makes it clearer that `set -e` is a distinct command by itself, and also indicates how we write multiple commands on separate lines which may be useful to anyone using this template repository.